fault/fault_test.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fault
+
+import (
+	"testing"
+
+	typev3 "github.com/envoyproxy/go-control-plane/envoy/type/v3"
+	"github.com/gogo/googleapis/google/rpc"
+)
+
+func TestErrorMessage(t *testing.T) {
+	tests := []struct {
+		desc                 string
+		fault                *AdapterFault
+		expectedErrorMessage string
+	}{
+		{
+			desc:                 "test0",
```

Comment:
The descriptions are better to be something more descriptive about what you are testing in this case :-)

Reply:
sure, updated.